test/conformance/testdata/ineligible_endpoints.yaml

```diff
@@ -295,3 +295,9 @@
 - endpoint: replaceNetworkingV1ServiceCIDRStatus
   reason: Kubernetes distribution would reasonably not allow this action via the API
   link: https://github.com/kubernetes/enhancements/pull/4983#issuecomment-2541699445
+- endpoint: connectCoreV1PostNamespacedPodExec
+  reason:  Unable to be tested
+  link: https://github.com/kubernetes/kubernetes/pull/133623#issuecomment-3229500806
```

Comment:
I think the original discussion in https://github.com/kubernetes/kubernetes/issues/112778 might be more insightful as to why it is inelligible?

It seems like the answer is not "unable to be tested" but rather "these are non-essential debug endpoints and we opted to exclude them"

(Which also implies we could revisit including them in the future, since debugging *is* generally pretty useful ...)

Reply:
Fixed up!